tools/generator/CodeGeneratorContext.cs

```diff
@@ -16,5 +16,6 @@ public class CodeGeneratorContext
 		string ContextMethodString => ContextMethod != null ? "in method " + ContextMethod.Name + " " : null;
 		string ContextTypeString => ContextType != null ? "in managed type " + ContextType.FullName : null;
 		public string ContextString => ContextFieldString + ContextMethodString + ContextTypeString;
+		public string ContextTypeMember => (ContextType != null ? $"{ContextType.FullName}." : string.Empty) + ContextMethod?.Name ?? ContextField?.Name;
```

Comment:
When `ContextMethod` isn't null, could this also include the parameter list?

The example in the PR message, `ColorStateList.CreateFromXml()`, contains *two* overloads.  It would presumably be helpful/useful if the warning message mentioned *which* overload was at fault:

    warning BG8800: Unknown return type 'System.Xml.XmlReader' for member 'Android.Content.Res.ColorStateList.CreateFromXml(Android.Content.Res.Resources, System.Xml.XmlReader)'.